packages/core/src/payment/strategies/braintree/is-us-bank-account-instrument-like.ts

```diff
@@ -0,0 +1,12 @@
+import { UsBankAccountInstrument } from '../../payment';
+
+export default function isUsBankAccountInstrumentLike(
+    instrument: any,
```

Comment:
Can you assign type for instrument

Reply:
it is unnecessary because passed type can be anything